routes/index.js

```diff
@@ -167,6 +188,22 @@ var routes = {
                 stripe_charge_create_service,
                 charge_id: charge.id
               });
+
+              basketData = {
+                last_name: customer.cards.data[0].name,
+                email: customer.email,
+                donation_amount: charge.amount,
+                currency: charge.currency,
+                created: charge.created,
+                recurring: false
+              };
+
+              if (ZERO_DECIMAL_CURRENCIES.indexOf(charge.currency) === -1) {
```

Comment:
I wonder if you can put this logic into a function/requirable file.

That way, on line 205, you can just do this:

```
basket.queue({
                last_name: customer.cards.data[0].name,
                email: customer.email,
                donation_amount: fixAmount(charge.amount, charge.currency),
                currency: charge.currency,
                created: charge.created,
                recurring: false
});
```